pkg/vault/contracts/authorizer/TimelockAuthorizer.sol

```diff
@@ -811,7 +884,14 @@ contract TimelockAuthorizer is IAuthorizer, IAuthentication, ReentrancyGuard {
         require(isRevoker(actionId, msg.sender, where), "SENDER_IS_NOT_REVOKER");
         bytes memory data = abi.encodeWithSelector(this.revokePermissions.selector, _ar(actionId), account, _ar(where));
         bytes32 revokePermissionId = getRevokePermissionActionId(actionId);
-        return _schedule(revokePermissionId, address(this), data, executors);
+        uint256 id = _schedule(revokePermissionId, address(this), data, executors);
+        // accounts that schedule actions are automatically made cancelers for them, so that they can manage their
+        // action. we check that they are not already a canceler since e.g. root may schedule actions (and root is
+        // always a global canceler).
```

Comment:
```suggestion
        // Revokers that schedule actions are automatically made cancelers for them, so that they can manage their
        // action. We check that they are not already a canceler since e.g. root may schedule revokes (and root is
        // always a global canceler).
```